app/assets/stylesheets/components/_typography.scss

```diff
@@ -14,6 +14,9 @@ body { -webkit-font-smoothing: antialiased; }
 .fs-12p { font-size: 12px; }
 .fs-20p { font-size: 20px; }
 
+.color-blue { color: $blue; }
+.color-navy { color: $navy; }
+
```

Comment:
Ah, I was trying to find the color classes, I thought they were ours, not basecss